src/main/charts/confluence/values.yaml

```diff
@@ -517,9 +567,9 @@ confluence:
     container: 
     
       requests:
-        # -- Initial CPU request by Confluence pod
-        #
-        cpu: "2" # -- If changing the cpu value update additional JVM arg 'ActiveProcessorCount' below
+        # -- Initial CPU request by Confluence pod.
+        # If changing the cpu value update additional JVM arg 'ActiveProcessorCount' below
+        cpu: "2"
```

Comment:
For consistency can we include a `#` above this property

Reply:
Fixed